tests/default.nf.test.snap

```diff
@@ -49,15 +49,17 @@
     },
     "Positive test of the Mobilome annotation pipeline with user proteins and virify input": {
         "content": [
-            23,
+            26,
             [
                 "pipeline_info",
                 "pos_test",
                 "pos_test/gff",
+                "pos_test/gff/pos_test.gff.gz",
+                "pos_test/gff/pos_test.gff.gz.tbi",
                 "pos_test/gff/pos_test_mobilome.gff.gz",
-                "pos_test/gff/pos_test_user_mobilome_clean.gff.gz",
-                "pos_test/gff/pos_test_user_mobilome_extra.gff.gz",
-                "pos_test/gff/pos_test_user_mobilome_full.gff.gz",
+                "pos_test/gff/pos_test_user_mobilome_clean.gff",
+                "pos_test/gff/pos_test_user_mobilome_extra.gff",
+                "pos_test/gff/pos_test_user_mobilome_full.gff",
```

Comment:
Probably is this file the one you want to index if we will publish for production the GFF file containing all the functional annotation generated by gene and pathways and systems in the ASA + mobilome.